Remove obsolete service message system commands

noplanman opened the pull request:
| ?            |  !
|---           | ---
| Type         | bug / improvement
| BC Break     | no
| Fixed issues | #994

#### Summary
Remove all obsolete service message system commands and instead try to execute any user-defined overrides directly in `GenericmessageCommand`.

#### Usage
You need to set `SystemCommand::$execute_deprecated = true;` before handling the request, either in your hook or getUpdates file.

If you have a custom `GenericmessageCommand` implementation, you'll need to add the code snippet from the original `GenericmessageCommand.php` to your own.

jacklul commented:
I don't think renaming `GenericmessageCommand` to `MessageCommand` is a good idea, might get confusing when people create `/message` commands or something

Assuming we allow users to use their own namespaces (#919) they should be able simply to extend this and `parent::execute()` instead of creating new class and copying code, this also prevents people from removing essential code for handling conversations.

jacklul commented on src/Commands/SystemCommands/GenericmessageCommand.php:
```diff
@@ -68,9 +68,57 @@ public function execute()
+        }
+
+        // Try to execute any deprecated system commands.
+        if ($deprecated_system_command_response = $this->executeDeprecatedSystemCommand()) {
```
I think `executeNoDb()` should call this as well?

noplanman replied:
Absolutely!

noplanman commented:
@jacklul I've updated this a bit and moved the code to `SystemCommand`, to make the integration into a custom `GenericmessageCommand` easier.

Also, there's a property `SystemCommand::$execute_deprecated` which needs to be set for this to work. I added this, to not have all the code run for users who don't need this feature.

Any thoughts are highly appreciated! 🙏

I'm also thinking about how best to implement the new command system, to make it logical and easy to use.
